### PR TITLE
🐛 fix: small improvements

### PR DESCRIPTION
**Pull request by DisaPadla**

### CODE REVIEW

**Comment by DisaPadla** [comment]
@iower @insulineru

**Review by insulineru — APPROVED** [review]
LGTM 👍🏻
Thanks for contributing, @DisaPadla. Good job!

**Comment by insulineru** [comment]
@DisaPadla need rebase, because we had new functional with separate wallet's states

**Comment by DisaPadla** [comment]
> LGTM 👍🏻 Thanks for contributing, @DisaPadla. Good job!

done

File: src/context/WalletProvider.tsx [hunk 1]
```diff
@@ -839,4 +839,4 @@ const WalletProvider = function WalletProvider({ children }: { children: React.R
   )
 }
 
-export { WalletProvider }
+export { WalletProvider }
```
Comment (insulineru) [inline src/context/WalletProvider.tsx]:
```suggestion
export { WalletProvider }

```

**Review by insulineru — APPROVED** [review]

**Pull request merged**